internal/parser/process.go

```diff
@@ -110,7 +123,7 @@ func ProcessEvent(e *EventPayload) *data.EffxEvent {
 			ServiceName:           &e.ServiceName,
 			Tags:                  &tagsPayload,
 			Actions:               &actions,
-			TimestampMilliseconds: timestampMilliseconds,
+			TimestampMilliseconds: &timestampMilliseconds,
```

Comment:
btw - we can remove any logic to make a default timestamp, the backend will now autoset the timestamp to time.Now() so clients don't need to.

Reply:
sure, it now only sets that field if the user set it explicitly.